com.microsoft.mrtk.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -489,24 +518,40 @@ private void RegisterPickupAction()
             interactable.OnClicked.AddListener(StartPlacement);
         }
 
+        /// <summary>
+        /// Registers the event which performs placement.
+        /// </summary>
+        private void RegisterPlacementEvent()
+        {
+            if (interactable == null)
```

Comment:
since `interactable` is a required component, a null check doesn't make sense.  This should be exceptional and cause an exception.